tests/app/test_jinja_filters.py

```diff
@@ -293,6 +293,24 @@ def test_format_household_member_name_trim_spaces(self):
 
         self.assertEqual(format_value, 'John Doe')
 
+    def test_format_household_member_name_possessive(self):
+        # Given
+        name = ['John', 'Doe']
+
+        # When
+        format_value = format_household_member_name_possessive(name)
+
+        self.assertEqual(format_value, 'John Doe\'s')
+
+    def test_format_household_member_name_possessive_trailing_s(self):
+        # Given
+        name = ['John', 'Does']
+
+        # When
+        format_value = format_household_member_name_possessive(name)
+
+        self.assertEqual(format_value, 'John Does\'')
+
```

Comment:
Should add a test for single name scenario also

Reply:
thats already covered by tests of `format_household_member_name`